contracts/types/Constants.sol

```diff
@@ -38,7 +38,7 @@ uint256 constant MODULE_TYPE_FALLBACK = 3;
 
 // Module type identifier for hooks
 uint256 constant MODULE_TYPE_HOOK = 4;
-bytes32 constant MODULE_ENABLE_MODE_TYPE_HASH = keccak256("ModuleEnableMode(address module, bytes32 initDataHash)");
+bytes32 constant MODULE_ENABLE_MODE_TYPE_HASH = keccak256("ModuleEnableMode(address module, uint256 moduleType, bytes32 userOpHash, bytes32 initDataHash)");
```

Comment:
According to the [[EIP-712 standard](https://eips.ethereum.org/EIPS/eip-712)](https://eips.ethereum.org/EIPS/eip-712), members are joined by a "," without any blank space.

`The type of a struct is encoded as name ‖ "(" ‖ member₁ ‖ "," ‖ member₂ ‖ "," ‖ … ‖ memberₙ ")" where each member is written as type ‖ " " ‖ name. For example, the above Mail struct is encoded as Mail(address from,address to,string contents).`

Check M-06 for more info.

Reply:
@filmakarov

Reply:
fixed!